src/utilities/report-utils.ts

```diff
@@ -74,7 +74,7 @@ export const getReportUrl = (questionRefId?: string) => {
     const classInfoUrl = portalData?.portalJWT?.class_info_url;
     const authDomainUrl = classInfoUrl?.split("/api")[0];
     const offeringBaseUrl = classInfoUrl?.split("/classes")[0]+"/offerings/";
-    const offeringId = portalData?.offering.id;
+    const offeringId = portalData?.offering?.id;
```

Comment:
@eireland has some code that deals with missing offering in a different way. So this might cause a conflict. But since this PR is older lets deal with the conflict in Evangeline's PR.